### PR TITLE
feat(wren-ui): Support record type in BigQuery

### PR DESCRIPTION
**Pull request by andreashimin**
## Description
Support record type in BigQuery

## UI Screenshots
image
image

### CODE REVIEW

**Review by fredalai — APPROVED** [review]
LGTM for frontend part.

File: wren-ui/migrations/20240928165009_add_model_nested_column.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
```
Comment (onlyjackfrost) [inline wren-ui/migrations/20240928165009_add_model_nested_column.js:5]: file name should be `{timestamp}_create_model_nested_column.js`

File: wren-ui/migrations/20240928165009_add_model_nested_column.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  return knex.schema.createTable('model_nested_column', (table) => {
+    table.increments('id').comment('ID');
+    table.integer('model_id').comment('Reference to model ID');
+    table.integer('column_id').comment('Reference to column ID');
```
Comment (onlyjackfrost) [inline wren-ui/migrations/20240928165009_add_model_nested_column.js:9]: The column_id was used in the service layer. 
If you would like to aggregate the column & nested column with the `column_path` and `model_id` (eg: mdlBuilder.ts file), then I think the column_id can be removed.
On the other hand, you can keep the column_id and use this column to aggregate. 
image

**Review by onlyjackfrost — COMMENTED** [review]
Besides the database schema and the usage, others LGTM

**Comment by andreashimin** [comment]
Hi @onlyjackfrost, I have renamed the migration file & replace the filter condition to `columnId` in https://github.com/Canner/WrenAI/pull/701/commits/de52d8f9ace27bc9ef57105f3dfaf60616f3885d.
Please kindly help review again when you have time, thanks :D

**Review by onlyjackfrost — APPROVED** [review]
LGTM

**Pull request merged**